tophat/opensource.tophat.com
# fix: Gh stars improvement

This PR uses the [github-metadata](https://github.com/jekyll/github-metadata) plugin to get the repositories star count. This fixes the issue where the projects are displayed in a different order first then re-ordered by stars.

# QA
 - Refresh the page and make sure it doesn't show the repositories out of order first.
